File: Assignment 2/if else/alphabet.cpp
#include<iostream>
using namespace std;
class Alphabet
{
char c;
    public:void alpha()
    {
        cout<<"Enter a character:\n";
        cin>>c;
        if(c>='a'&&c<='z'||c>='A'&&c<='Z')
        {
            cout<<"The character is an alphabet";
        }
        else{cout<<"The character is not an alphabet";}
        }
    };

int main()
{
    Alphabet obj;
    obj.alpha();
}

File: Java assignment(1-2)/java Assignment 2/Alphabet.java
import java.util.*;
class Alphabet
{
char alpha;
void accept()
{
Scanner sc=new Scanner(System.in);
System.out.println("Enter a character: ");
alpha=sc.next().charAt(0);
if(alpha>='A' && alpha<='Z' || alpha>='a' && alpha<='z')
{
System.out.println("The Entered character is an Alphabet");
}
else
{
System.out.println("The Entered character is not an Alphabet");
}
}
public static void main(String...args)
{
Alphabet obj=new Alphabet();
obj.accept();
}
}
